inc/dbmysql.class.php

```diff
@@ -261,130 +237,116 @@ function result($result, $i, $field) {
       return NULL;
    }
 
-
    /**
-    * Give number of rows of a Mysql result
-    *
-    * @param $result MySQL result handler
-    *
-    * @return number of rows
-   **/
+    * Number of rows
+    * @param mysqli_result $result MySQL result handler
+    * @return int number of rows
+    */
    function numrows($result) {
       return $result->num_rows;
    }
 
-
    /**
-    * Fetch array of the next row of a Mysql query
+    * fetch_array() of the next row of a Mysql query
```

Comment:
@trasher Coul you, please, give a little more feedback ?
Problem is due to lower case, function name usage, another things ?